scrapy/extensions/feedexport.py

```diff
@@ -47,7 +47,16 @@ def store(file):
 class BlockingFeedStorage(object):
 
     def open(self, spider):
-        return TemporaryFile(prefix='feed-')
+        folder = None
+        path = spider.crawler.settings['BLOCKING_FEED_STORAGE_PATH']
```

Comment:
you can probably use `path` all along, and not use `folder` at all,
something like
```
        path = spider.crawler.settings.get('BLOCKING_FEED_STORAGE_PATH')
        if path:
            if not os.path.isdir(path):
                raise OSError('Not a Directory: ' + str(path))

        return NamedTemporaryFile(prefix='feed-', dir=path)
```